sdk/vaa/structs_test.go

```diff
@@ -108,6 +109,32 @@ func TestAddress_MarshalJSON(t *testing.T) {
 	assert.NoError(t, err)
 }
 
+func TestAddress_UnmarshalJSON(t *testing.T) {
+	addr, _ := StringToAddress("0x0290fb167208af455bb137780163b7b7a9a10c16")
+
+	b, err := addr.MarshalJSON()
+	require.NoError(t, err)
+
+	var unmarshalAddr Address
+	err = unmarshalAddr.UnmarshalJSON(b)
+	require.NoError(t, err)
+
+	assert.Equal(t, addr, unmarshalAddr)
+}
+
+func TestAddress_Unmarshal(t *testing.T) {
+	addr, _ := StringToAddress("0x0290fb167208af455bb137780163b7b7a9a10c16")
+
+	b, err := json.Marshal(addr)
+	require.NoError(t, err)
+
+	var unmarshalAddr Address
+	err = json.Unmarshal(b, &unmarshalAddr)
+	require.NoError(t, err)
+
+	assert.Equal(t, addr, unmarshalAddr)
+}
+
```

Comment:
Would also be good to add tests for cases where the string is too short or too long.

Reply:
Added tests.